6-jre7/Dockerfile

```diff
@@ -34,5 +34,18 @@ RUN set -x \
 	&& rm bin/*.bat \
 	&& rm tomcat.tar.gz*
 
+RUN set -x \
+	&& apt-get update \
+	&& apt-get install -yq gcc make openssl libssl-dev libapr1 libapr1-dev openjdk-7-jdk="$JAVA_DEBIAN_VERSION" \
```

Comment:
Could you sort the list and put it in a variable so it is easier to purge later? It should make it more obvious that `libapr1` is not a build dependency, but a run dependency.  Redis has a pretty similar example: https://github.com/docker-library/redis/blob/98a586a0f5c505e120b22dc94efc1fa5c492ce35/3.0/Dockerfile#L24-L36.  You should also drop `openssl` since it is already in the java base image since it has curl.